js/index.js

```diff
@@ -9,12 +9,53 @@ copyright.innerHTML = `Diana Carachure ${thisYear}`;
 
 footer.appendChild(copyright);
 
-const skills = ['JavaScript', 'HTML', 'CSS'];
+const skills = ['JavaScript', 'HTML', 'CSS', 'Access', 'Excel','Data Entry', 'Apricot by Social Solutions'];
 const skillsSection = document.getElementById('skills');
 const skillsList = skillsSection.querySelector('ul'); //replace document with skillsSection -only "looking" in that section
 
-for (let i = 0; i < skills; i++){
-    const skill = document.createElement('li');
-    skill.innerHTML = `${skills}`;
+for (let i = 0; i < skills.length; i++){
+    let skill = document.createElement('li');
+    skill.innerText = skills[i];
     skillsList.appendChild(skill);
-};
\ No newline at end of file
+};
+
+//code below will handle the messages submitted in the form section of the page
+let messageForm = document.getElementsByName("leave_message");
```

Comment:
```suggestion
let messageForm = document.getElementsByName("leave_message")[0];
```
This function returns an array of all the elements with that name, you need to get the first (and only) one of those to get the element itself